src/fromager/wheels.py

```diff
@@ -111,7 +111,7 @@ def build_wheel(
     wheels = list(ctx.wheels_build.glob("*.whl"))
     if wheels:
         return wheels[0]
-    return None
+    raise FileNotFoundError(f"Could not locate the built wheels for {req.name}")
```

Comment:
@dhellmann from what I noticed wherever this function is called, the caller already expects a non null return value. Do you think if we raise an error instead of returning null it would be fine? Or would you rather make this change later?

Reply:
Let's do that separately so we can focus on test coverage for it.

Reply:
okay then I will fix the type errors from this separately